src/podman-cli.ts

```diff
@@ -0,0 +1,106 @@
+/**********************************************************************
+ * Copyright (C) 2022 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+import { isMac, isWindows } from './util';
```

Comment:
if it needs to stay even after podman 5.0 I think you should not have duplicates of the podman extension code there

probably we would need to see how podman extension can export an API that this extension could consume

Reply:
moved conversation to [#35](https://github.com/redhat-developer/podman-desktop-redhat-account-ext/pull/35/files#r1487218477)

Reply:
tsconfig.json fixed and contains only required changes https://github.com/redhat-developer/podman-desktop-redhat-account-ext/pull/35/files#r1487228251